q2_phylogenomics/plugin_setup.py

```diff
@@ -500,13 +500,21 @@
     'pileups': 'The PileUp data.'
 }
 
+consensus_sequence_parameter_descriptions = {
+    'min_depth': 'The minimum depth to call a consensus base.'
+}
+
+consensus_sequence_parameters = {
+    'min_depth': Int
+}
+
 plugin.methods.register_function(
     function=q2_phylogenomics._assemble.consensus_sequence,
     inputs={'pileups': SampleData[PileUp]},
-    parameters={},
+    parameters=consensus_sequence_parameters,
     outputs=consensus_sequence_outputs,
     input_descriptions=consensus_sequence_input_descriptions,
-    parameter_descriptions={},
+    parameter_descriptions=consensus_sequence_parameter_descriptions,
```

Comment:
should this be exposed in the pipeline? if so, this PR would be a good place to add it.